### DISCUSSION TITLE
闲聊：PCL 匹配上了一个 Sigma 规则（没说 PCL 是病毒）

### DISCUSSION THREAD

**Discussion by tangge233** [Meloong-Git/PCL#4673] [讨论]
把 PCL 上传到云端分析发现
![image](https://github.com/user-attachments/assets/722731ad-c32c-4bd4-8f45-32205a865a4c)
这个描述是
![image](https://github.com/user-attachments/assets/c77d8559-eb7a-47a5-bfe8-dbe06efb11a4)
这个程序是
![image](https://github.com/user-attachments/assets/0f5a2e67-bc4a-4ea0-85b0-b8d3ea6afad5)
就很奇怪。
莫非是被误报的原因之一？

**Comment by LTCatt** [comment]
这应该是沙盒的误检测，沙盒的 bug

**Reply by LTCatt** [reply]
Sigma 规则应该不是现在报毒的原因，之前 PCL 极客蓝解密里倒序的网址（:sptth）也被 Sigma 匹配上了，但修改之后还是报毒。
Windows Defender 现在的报毒是 AI 检测导致的。
